server/poll/transform.go

```diff
@@ -113,6 +117,11 @@ func (p *Poll) makeAdditionalText(localizer *i18n.Localizer, numberOfVotes int)
 	}
 
 	lines := []string{"---"}
+
+	if p.Settings.Progress {
+		lines = append(lines, generateProgressBars(votes, numberOfVotes)...)
```

Comment:
I prefer this line below "TotalVotes" line. [0/5](https://docs.mattermost.com/process/training.html#list-of-terms)